fre/pp/ppyaml_test/pp.yaml

```diff
@@ -1,225 +1,79 @@
-## User defined edits
-# Paths for where the rose-suite and rose-app configurations are in checked out pp repo
```

Comment:
We need to also make sure that changes here are reflected in the schema.json so the pp.yaml can be validated correctly

Reply:
Thank you, I hadn't even thought of that. The good news is I see the "yaml valid" message each time during my testing so it may be ok this case?

Reply:
True but we should definitely check. There is extra information in the schema.json now (since the updated pp.yaml would not have the other environment variables at the top)

Reply:
And we should make sure everything is defined how it's supposed to be, either required or null, yadda yadda

Reply:
You were completely right- the schema needed some corresponding updates. I think I made them now. However, I'm still not sure about the user-defined variables (the new "FRE properties").

I removed all except &xyInterp, but it's not in schema and still seems to work. Must the user variable names be in the schema? I think it would be better if they didn't.

Reply:
Thanks you're right. I removed the rose-app and suite filepath configurations from the schema